src/FINDOLOGIC/Api/Responses/Xml21/Properties/Filter.php

```diff
@@ -0,0 +1,148 @@
+<?php
+
+namespace FINDOLOGIC\Api\Responses\Xml21\Properties;
+
+use FINDOLOGIC\Api\Helpers\ResponseHelper;
+use SimpleXMLElement;
+
+class Filter
+{
+    /** @var int|null $itemCount */
+    private $itemCount;
+
+    /** @var string|null $cssClass */
+    private $cssClass;
+
+    /** @var string|null $noAvailableFiltersText */
+    private $noAvailableFiltersText;
+
+    /** @var string $name */
+    private $name;
+
+    /** @var string $display */
+    private $display;
+
+    /** @var string $select */
+    private $select;
+
+    /** @var int $selectedItems */
+    private $selectedItems = 0;
+
+    /** @var string $type */
+    private $type;
+
+    /** @var Attributes|null $attributes */
+    private $attributes;
+
+    /** @var Item[] $items */
+    private $items = [];
+
+    public function __construct(SimpleXMLElement $response)
+    {
+        $this->itemCount = ResponseHelper::getIntProperty($response, 'itemCount', true);
+        $this->cssClass = ResponseHelper::getStringProperty($response, 'cssClass');
+        $this->noAvailableFiltersText = ResponseHelper::getStringProperty($response, 'noAvailableFiltersText');
+        $this->name = ResponseHelper::getStringProperty($response, 'name');
+        $this->display = ResponseHelper::getStringProperty($response, 'display');
+        $this->select = ResponseHelper::getStringProperty($response, 'select');
+        $this->type = ResponseHelper::getStringProperty($response, 'type');
+
+        if ($value = ResponseHelper::getIntProperty($response, 'selectedItems')) {
```

Comment:
How about `ResponseHelper::getIntProperty($response, 'selectedItems')) ?: 0`?

Reply:
Thanks, never used that operator before